distribution/install.sh

```diff
@@ -207,6 +207,29 @@ add_to_path() {
 # be updated if necessary.
 # ------------------------------------------------------------------------------
 
+get_gnu_musl_glibc() {
+  need_cmd ldd
+  need_cmd bc
+  need_cmd awk
+  # Detect both gnu and musl
+  # Also detect glibc versions older than 2.18 and return musl for these
+  local _ldd_version
+  local _glibc_version
+  _ldd_version=$(ldd --version)
+  if [[ $_ldd_version =~ "GNU" ]]; then
+    _glibc_version=$(echo "$_ldd_version" | awk '/ldd/{print $NF}')
+    if [ 1 -eq "$(echo "${_glibc_version} < 2.18" | bc)" ]; then
+      echo "musl"
+    else
+      echo "gnu"
+    fi
+elif [[ $_ldd_version =~ "musl" ]]; then
+  echo "musl"
+else
+  err "Unknown architecture from ldd"
```

Comment:
Maybe we could print out the ldd version output here? That might help with bug reports at least.